lib/clients/github.go

```diff
@@ -167,7 +171,12 @@ func (g realGHClient) request(f func() (interface{}, *github.Response, error)) (
 	b := backoff.NewExponentialBackOff()
 	b.MaxElapsedTime = g.config.GetTimeout()
 
-	er := backoff.Retry(op, b)
+	er := backoff.RetryNotify(op, b, func(err error, duration time.Duration) {
+		duration /= 1000000 // Convert nanoseconds to milliseconds
+		duration *= 1000000 // Convert back so it appears correct
```

Comment:
Right, but the numbers are randomized. I was instead getting things like `1s374.926427ms`, so I wanted to truncate it to millisecond-level precision to make it more readable.